Jenkinsfile

```diff
@@ -1,15 +1,22 @@
-@Library('socrata-pipeline-library@3.0.0') _
+@Library('socrata-pipeline-library@6.0.0') _
 
 commonPipeline(
     defaultBuildWorker: 'build-worker',
     jobName: 'geo-export',
     language: 'scala',
-    languageVersion: '2.12',
+    languageOptions: [
+	    crossCompile: true,
```

Comment:
Why cross compile?

Reply:
Tabs switched to spaces.

Also, we use cross-compile because these refactoring intends to be at parity with the system being replaced by the refactoring, sbtBuild.

[Logs](https://jenkins-build.socrata.com/job/geo-export-release/63/console) indicate that cross-compilation is currently being used:
```
Executing sbt  +clean +compile  test
```